script/railcar

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/env node
+
+var fs = require('fs')
+var exec = require('child_process').exec
+var async = require('../build/node_modules/async')
+
+var branchName, leadingBranchName
+
+async.series([
+  parseArguments,
+  checkCleanWorkingTree,
+  checkoutBranch,
+  pullUpstreamChanges,
+  mergeLeadingBranch,
+  fetchUpstreamTags,
+  updateVersionNumber,
+  pushLocalChanges,
+  pushLocalTags,
+], finish)
+
+function parseArguments(next) {
+  branchName = process.argv[2]
+  switch (branchName) {
+    case 'beta':
+      leadingBranchName = 'master'
+      break
+    case 'stable':
+      leadingBranchName = 'beta'
+      break
+    default:
+      return next(new Error('Usage: script/railcar <beta|stable>'))
+  }
+  next()
+}
+
+function checkCleanWorkingTree(next) {
+  console.log('Checking for a clean working tree');
+  exec('git status --porcelain', function(error, output) {
```

Comment:
Is the `error` here handled? It looks like `next` might be being called with `null` even when `error` is present since `output` will be an empty string.